libraries/shared/src/ShapeInfo.cpp

```diff
@@ -115,12 +115,6 @@ void ShapeInfo::setParams(ShapeType type, const glm::vec3& halfExtents, QString
         case SHAPE_TYPE_HULL:
         case SHAPE_TYPE_MULTISPHERE:
             break;
-        case SHAPE_TYPE_SPHERE: {
-                float radius = glm::length(halfExtents) / SQUARE_ROOT_OF_3;
-                radius = glm::max(radius, MIN_HALF_EXTENT);
-                _halfExtents = glm::vec3(radius);
```

Comment:
the halfExtents are already set above, and we want to pass the actual halfExtents through so that the collision hull code can determine if we're actually a sphere or if we're an ellipsoid